src/store/ofac.ts

```diff
@@ -62,6 +62,7 @@ export const ofacApi = createApi({
           return { error: { status: 'CUSTOM_ERROR', data: (error as Error).message } }
         }
       },
+      keepUnusedDataFor: 24 * 60 * 60, // 2
```

Comment:
Outdated comment (should be 1 day?)

Reply:
No, I somehow managed to delete the "4 hours" from "24 hours" :D